README.md

```diff
@@ -1,556 +1,53 @@
-[![CI](https://github.com/f3d-app/f3d/actions/workflows/ci.yml/badge.svg)](https://github.com/f3d-app/f3d/actions/workflows/ci.yml) [![codecov](https://codecov.io/gh/f3d-app/f3d/branch/master/graph/badge.svg?token=siwG82IXK7)](https://codecov.io/gh/f3d-app/f3d) [![Downloads](https://img.shields.io/github/downloads/f3d-app/f3d/total.svg)](https://github.com/f3d-app/f3d/releases) [![Downloads](https://img.shields.io/reddit/subreddit-subscribers/f3d_app.svg)](https://www.reddit.com/r/f3d_app)
+---
+layout: default
+title: Home
+nav_order: 0
+description: "F3D - Fast and minimalist 3D viewer"
+permalink: /
+---
 
-![F3D Logo](./resources/logo.svg)
+[![CI](https://github.com/f3d-app/f3d/actions/workflows/ci.yml/badge.svg)](https://github.com/f3d-app/f3d/actions/workflows/ci.yml) [![codecov](https://codecov.io/gh/f3d-app/f3d/branch/master/graph/badge.svg?token=siwG82IXK7)](https://codecov.io/gh/f3d-app/f3d) [![Downloads](https://img.shields.io/github/downloads/f3d-app/f3d/total.svg)](https://github.com/f3d-app/f3d/releases) [![Downloads](https://img.shields.io/reddit/subreddit-subscribers/f3d_app.svg)](https://www.reddit.com/r/f3d_app)
 
 # F3D - Fast and minimalist 3D viewer
-
 By Michael Migliore and Mathieu Westphal.
 
-F3D (pronounced `/fɛd/`) is a [VTK-based](https://vtk.org) 3D viewer following the [KISS principle](https://en.wikipedia.org/wiki/KISS_principle), so it is minimalist, efficient, has no GUI, has simple interaction mechanisms and is fully controllable using arguments on the command line.
+<img src="https://user-images.githubusercontent.com/3129530/195972584-11ebb000-4939-47f4-9719-9d55e18867b0.png" align="left" width="20px"/>
```

Comment:
This doesnt look superb, either on github or on jekyll.

The problem is that I did not want to keep the big logo in the middle, it takes to much space and is already visible top right and favicon in the website. But I still wanted to keep it visible somewhere in github, hence the small logo inside the text. Should we keep it or drop it ? If we drop it, where should we put a logo ? @Meakk

Reply:
I think it looks fine even on github despite the tabulation.

Reply:
ok